spec/services/nytimes_service_spec.rb

```diff
@@ -9,22 +9,24 @@
 			end
 
 			expect(Article.first.title).to(
-				eq("California Imposes Water Restrictions, in First for State")
+				eq("A Nuclear Deal Built on Coffee, Compromise and No Sleep")
 			)
 			expect(Article.first.url).to(
-				eq("http://www.nytimes.com/2015/04/02/us/california-imposes-first-" +
-					 "ever-water-restrictions-to-deal-with-drought.html")
+				eq("http://www.nytimes.com/2015/04/04/world/middleeast/" +
```

Comment:
Indent the first parameter one step more than the previous line.<br>Tab detected.